azerothcore/azerothcore-wotlk
# fix(DB/GameObject): Fixes reapawn timer for Waterlogged Letter

Decreased the spawn timer to 10 minutes

Fix by https://github.com/JamesGent 
I just created the PR



## Changes Proposed:
-  Decreased the spawn timer from 2 hours to 10 minutes

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7021

## SOURCE:

https://wotlk.evowow.com/?object=2656#contains

https://classic.wowhead.com/quest=637/sully-balloos-letter

## Tests Performed:

- Launched the SQL query and checked the respawn timer had updated

## How to Test the Changes:


1. Go to arathi under thandol span
2. Find the dwarf body crushed
3. Check letter spawn time


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
